src/web/core/src/main/java/org/geoserver/web/data/resource/AttributeTypeInfoEditPanel.java

```diff
@@ -0,0 +1,473 @@
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```

Comment:
I expect you were just copying an example file, or if this is new I can be 2025

Reply:
Fixed. Thanks!